io.playnux.calculator.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+app-id: io.playnux.calculator
```

Comment:
`playnux.io` doesn't seem to exist. This make the app-id no meeting the requirements.

Reply:
you didn't answer this (marking as resolved doesn't count)